didx-xyz/yoma-mobile
# YOMA-473 - chore: adding sentry

## Scope [YOMA-473](https://yomaworld.atlassian.net/browse/YOMA-473)

![](https://media.giphy.com/media/3oEjHUf7j0aFDce0dG/giphy.gif)

## Work Done
- adding Sentry
- adding a new staging env for Android builds into Firebase App Deployment
- adding a handful of docs changes (probably needs to be filled out in a separate task)

## Steps to Test
have access to sentry, make the app throw an error:
```ts
throw new Error("My first Sentry error!");
```
and see that it's pulling through into Sentry.
